Automattic/wp-calypso
# Hosting Command Palette: Update the highlighted color to match the accent color of the dashboard color scheme

Related to https://github.com/Automattic/dotcom-forge/issues/4496

## Proposed Changes

This PR changes the color of the items on the command palette available on the `/sites` page to match the color of the **Dashboard color scheme** that is set on https://wordpress.com/me/account

<img width="1712" alt="Screenshot 2023-11-15 at 11 16 50 AM" src="https://github.com/Automattic/wp-calypso/assets/25575134/9374909c-e1cf-4942-b1b0-f511c2f7eca2">

## Testing Instructions

* Pull changes from this PR or navigate to Calypso live link
* Navigate to `/sites` page
* Press `cmd + k` on Mac or `ctr + k` on Windows
* Observe that the command palette color matches the accent color of the dashboard theme and is consistent with the dashboard color scheme selected:

<img width="1722" alt="Screenshot 2023-11-15 at 11 16 20 AM" src="https://github.com/Automattic/wp-calypso/assets/25575134/9a09a5a1-db00-4dcc-a813-20f1cf6ec4ea">

* Navigate to `https://wordpress.com/me/account`
* Select a couple of different themes and confirm the command palette colors are consistent with the theme selected and with colors in the Gutenberg command palette:

<img width="1712" alt="Screenshot 2023-11-15 at 11 16 50 AM" src="https://github.com/Automattic/wp-calypso/assets/25575134/8ec35084-0fb3-4a59-affa-d9d837213798">




## Pre-merge Checklist
